src/app/Pages/ImageDetails.tsx

```diff
@@ -0,0 +1,46 @@
+import * as React from 'react'
+import {
+   PageSection, Bullseye, Text, TextContent, TextVariants,
+  Card, CardTitle, CardBody, CardFooter
+} from '@patternfly/react-core'
+import { useDocumentTitle } from '@app/utils/useDocumentTitle'
+import Footer from '@app/components/Footer'
+import DetailsView from '@app/components/DetailsView'
+import { useState, useEffect } from 'react'
+import { useParams } from 'react-router-dom'
+
+const ImageDetails: React.FunctionComponent<{title: string}> = ({title}) => {
+  const [details, setDetails] = useState({})
+  let { provider, region, imageName } = useParams()
```

Comment:
Same minor thing here :slightly_smiling_face:,  if `provider`, `region` and `imageName` won't be reassigned, it is recommended to use `const` when using [eslint/prefer-const](https://eslint.org/docs/latest/rules/prefer-const)